components/tag/Tag.js

```diff
@@ -42,9 +43,13 @@ class Tag extends PureComponent {
         {onLabelClick ? (
           <Button className={theme['label-button']} onClick={onLabelClick} level="outline" inverse={inverse}>
             {label}
+            {children}
           </Button>
         ) : (
-          <span className={theme['label']}>{label}</span>
+          <span className={theme['label']}>
+            {label}
+            {children}
```

Comment:
Q: What if label or children now isn't provided? I'm not entirely sure what will happen then... Shouldn't there be checks now?

Reply:
**A:** I removed the `label` prop and made `children` required. In this case we don't need an extra check. A console warning will be thrown.

 :ok: by @driesd on 20/02/2018 17:05:11